test/langtools/tools/javac/valhalla/lworld-values/ConflictingSuperInterfaceTest.java

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Missing Copyright.

Reply:
This is intentional. In javac's negative tests we don't add a copyright, but a tag that says nodynamiccopyright. 

I have forgotten the reason we do that, but that is the right behavior :)